drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
Yeah, good point. I'd probably move all of those discussions to a separate Performance document though. We can then simply refer it here. Something like

> In typical usage scenarios, the API documentation needs to be used frequently and should thus [be cached] (link to Performance).